src/app/home/reminder.service.ts

```diff
@@ -1,105 +1,107 @@
 import { Injectable } from '@angular/core';
 import { Storage } from '@ionic/storage';
 
+
+import { db } from 'src/environments/environment';
+import { getDocs, collection, query, where, deleteDoc, doc, updateDoc } from 'firebase/firestore'
+import { getAuth } from "firebase/auth";
+
 @Injectable({
   providedIn: 'root',
 })
 
 export class ReminderService {
   constructor(private storage: Storage) {
```

Comment:
I think it would be best to remove the created ionic storage from this constructor since it looks like you are currently using firebase for the database.